bin/dump_kubernetes.sh

```diff
@@ -73,7 +84,64 @@ dump_time() {
   date -u > "${OUT_DIR}/DUMP_TIME"
 }
 
-dump_logs() {
+dump_logs_for_container() {
+  local namespace="${1}"
+  local pod="${2}"
+  local container="${3}"
+
+  log "Retrieving logs for ${namespace}/${pod}/${container}"
+
+  mkdir -p "${LOG_DIR}/${namespace}/${pod}"
+  local log_file_head="${LOG_DIR}/${namespace}/${pod}/${container}"
+
+  local log_file="${log_file_head}.log"
+  kubectl logs --namespace="${namespace}" "${pod}" "${container}" \
+      > "${log_file}"
+
+  local filter="?(@.name == \"${container}\")"
+  local json_path='{.status.containerStatuses['${filter}'].restartCount}'
+  local restart_count
+  restart_count=$(kubectl get --namespace="${namespace}" \
+      pod "${pod}" -o=jsonpath="${json_path}")
+  if [ "${restart_count}" -gt 0 ]; then
+    log "Retrieving previous logs for ${namespace}/${pod}/${container}"
+
+    local log_previous_file
+    log_previous_file="${log_file_head}_previous.log"
+    kubectl logs --namespace="${namespace}" \
+        --previous "${pod}" "${container}" \
+        > "${log_previous_file}"
+  fi
+}
+
+copy_core_dumps_if_istio_proxy() {
+  local namespace="${1}"
+  local pod="${2}"
+  local container="${3}"
+
+  if [ "istio-proxy" = "${container}" ]; then
+    local out_dir="${LOG_DIR}/${namespace}/${pod}"
+    mkdir -p "${out_dir}"
+    local core_dumps
+    core_dumps=$(kubectl exec -n "${namespace}" "${pod}" -c "${container}" -- \
+        find /etc/istio/proxy -name 'core.*')
```

Comment:
please also add the new path /var/istio/proxy

Reply:
Done.